tests/Integration/Behaviour/Features/Scenario/ApiAccess/api_access_management.feature

```diff
@@ -146,6 +165,7 @@ Feature: Api Access Management
       | enabled     | true                                   |
       | description | may the force                          |
       | scopes      | api_access_read, hook_read, hook_write |
+      | lifetime    | 3600                                   |
```

Comment:
Can you vary the value a little bit please, to make sure it's really dynamic Especially since the default value is already 3600